dsl/properties/scripts/DockerClient.groovy

```diff
@@ -851,4 +853,110 @@ public class DockerClient extends BaseClient {
             dockerClient.connectNetwork(networkList[i], container)
         }  
     }
-}
\ No newline at end of file
+
+    def createIngress(def subnetList,
+                         def gatewayList,
+                         def enableIpv6,
+                         def attachable,
+                         def mtu,
+                         def labels){
+
+        if(!standAloneDockerHost()){
+
+            def networkName = "ingress"
```

Comment:
As per https://docs.docker.com/engine/swarm/networking/#customize-the-ingress-network, the ingress network can be named something else other than 'ingress' based on a user-specified value but there can only be one ingress network. The procedure logic should account for it.